pkg/builtin/manifest.yml

```diff
@@ -298,6 +298,50 @@ extensions:
             - id: title
               type: string
               title: Title
+            - id: showTitle
+              type: bool
+              title: Show Title
+            - id: position
+              type: string
+              title: Position
+              defaultValue: right
+              choices:
+                - key: right
+                  label: Right
+                - key: middle
+                  label: Middle
+                - key: left
+                  label: Left
+            - id: size
+              type: string
+              title: Width
+              defaultValue: small
+              choices:
+                - key: small
+                  label: Small
+                - key: medium
+                  label: Medium
+                - key: large
+                  label: Large
+            - id: heightType
+              type: string
+              title: Height Type
+              defaultValue: auto
+              choices:
+                - key: auto
+                  label: Auto
+                - key: manual
+                  label: Manual
+            - id: height
+              type: number
+              title: Height
+              #              min: 0
+              #              max: 70
```

Comment:
Is there any reason why you left the comment here?

Reply:
@HideBa red said that he will provide me the min/max later ... that's why I guess
let me double-check with him and decide what to do with those two lines